Runtimes/Unity/BepInEx.Unity.IL2CPP/Il2CppInteropManager.cs

```diff
@@ -120,14 +122,9 @@ static void HashString(ICryptoTransform hash, string str)
         }
 
         HashFile(md5, GameAssemblyPath);
-
-        if (Directory.Exists(UnityBaseLibsDirectory))
-            foreach (var file in Directory.EnumerateFiles(UnityBaseLibsDirectory, "*.dll",
-                                                          SearchOption.TopDirectoryOnly))
-            {
-                HashString(md5, Path.GetFileName(file));
-                HashFile(md5, file);
-            }
```

Comment:
The hash check of base libraries should stay just in case they get corrupted for whatever reason.